marketplace/n8n.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "./schema.json",
+  "code": "n8n",
+  "name": "n8n",
+  "description": "Build complex automations 10x faster, without fighting APIs",
+  "icon": "https://raw.githubusercontent.com/n8n-io/n8n/master/assets/n8n-logo.png",
+  "image": "n8nio/n8n",
+  "command": null,
+  "args": null,
+  "port": 5678,
+  "network": "HTTP",
+  "dataPath": "/home/node/.n8n",
+  "variables": [
+    {
+      "name": "DB_TYPE",
+      "value": "postgresdb"
+    },
+    {
+      "name": "DB_POSTGRESDB_USER",
+      "value": "${POSTGRESQL_USERNAME}"
+    },
+    {
+      "name": "DB_POSTGRESDB_PASSWORD",
+      "value": "${POSTGRESQL_PASSWORD}"
+    },
+    {
+      "name": "DB_POSTGRESDB_HOST",
+      "value": "${POSTGRESQL_HOST}"
+    },
+    {
+      "name": "DB_POSTGRESDB_PORT",
+      "value": "${POSTGRESQL_PORT}"
+    },
```

Comment:
Internally it is `${POSTGRES_*}` rather than `${POSTGRESQL_*}`

<img width="391" alt="image" src="https://github.com/zeabur/zeabur/assets/28441561/7d9dc419-4e9e-4120-9c79-d0f4c8a5efd0">

Reply:
In my case, it seems that there aren't so many variables.
<img width="1522" alt="image" src="https://github.com/zeabur/zeabur/assets/19288423/85e03e4e-1800-403f-8f32-f62e386e793c">

I add this by the zeabur docs: https://zeabur.com/docs/databases/postgresql. And i will change it.

Reply:
Done @pan93412